src/main/java/com/anysoftkeyboard/languagepack/tamazight/PackBroadcastReceiver.java

```diff
@@ -1,14 +1,14 @@
-package com.anysoftkeyboard.languagepack.languagepack;
-
-import android.content.BroadcastReceiver;
-import android.content.Context;
-import android.content.Intent;
-
-public class PackBroadcastReceiver extends BroadcastReceiver {
-
-    @Override
-    public void onReceive(Context arg0, Intent arg1) {
-
-    }
-
-}
+package com.anysoftkeyboard.languagepack.languagepack;
```

Comment:
This should be `package com.anysoftkeyboard.languagepack.tamazight;`

Reply:
make sure you change the reference in `AndroidManifest.xml`, as well.
here: https://github.com/BoFFire/LanguagePack/blob/da4a3f8cf8a41ec60cb4b8d9c416eb1676a4daeb/src/main/AndroidManifest.xml#L3
and here: https://github.com/BoFFire/LanguagePack/blob/da4a3f8cf8a41ec60cb4b8d9c416eb1676a4daeb/src/main/AndroidManifest.xml#L17